docs/DeveloperGuide.md

```diff
@@ -1,38 +1,275 @@
+
+### Recipe Possible Command
+
+The recipe possible command is facilitated by `RecipePossibleCommand`. 
+
+It requires `RecipeList` and `IngredientList` of `MealCompanionSession`.
+
+Given below is the only example usage scenario and how the recipe possible command behaves at each step.
+
+Step 1: User wants to get a list of recipes that can be made with the current list of ingredients. User calls `recipe possible`.
+
+Step 2: `RecipePossibleCommand` executes by retrieving the `RecipeList` and `IngredientList` of `MealCompanionSession`.
+
+Step 3: Every `Recipe` in `RecipeList` is checked against all `Ingredient` in `IngredientList` to see if it can be made.
+
+Step 4: `Recipe` that can be made are outputted.
+
+The following sequence diagram shows how the Recipe Possible Command works:
+
+![RecipePossibleCommandSequence.png](images%2FRecipePossibleCommandSequence.png)
+
+###### [Back to table of contents](#table-of-contents)
+
+### Recipe All Command
+
+The recipe possible command is facilitated by `RecipeAllCommand`.
+
+It requires `RecipeList` of `MealCompanionSession`.
+
+Given below is the only example usage scenario and how the recipe possible command behaves at each step.
+
+Step 1: User wants to get the full list of recipes stored in the program. User calls `recipe all`.
+
+Step 2: `RecipeAllCommand` executes by retrieving the `RecipeList` of `MealCompanionSession`.
+
+Step 3: Every `Recipe` in `RecipeList` is outputted.
+
+The following sequence diagram shows how the Recipe All Command works:
+
+![RecipeAllCommandSequence.png](images%2FRecipeAllCommandSequence.png)
+
+###### [Back to table of contents](#table-of-contents)
+
+## Documentation, logging, testing, configuration, dev-ops
+
+{insert Documentation guide link}
+{insert Testing guide link}
+{insert Logging guide link}
+{insert Configuration guide link}
+{insert DevOps guide link}
+
+## Appendix: Requirements
+
+### Product scope
+
+#### Target user profile
+Our target user profile is the average School of Computing student that resides on campus, regardless of Residential College (RC), Halls, Residences or Houses. There is a focus on those who regularly cook in their hostels or would like to pick up cooking.
+
+#### Value proposition
+Our product targets students who would like to save money on buying or ordering food, and also to save time by preparing their meals beforehand, so they can eat immediately after a class without having to queue for food or look for a seat in a crowded canteen on campus. Aside from RCs, hostels on campus do not provide meal plans, so our product addresses this by helping users pick up cooking and manage the ingredients they have in their hostel refrigerators. Even within RCs, the meal plans do not cover lunch, meaning students need to source their own lunch, which is a problem that our product also aims to alleviate.
+
+###### [Back to table of contents](#table-of-contents)
+
+### User Stories
+
+#### v1.0
```

Comment:
It was helpful that V1.0 and V2.0 user stories were written in two separate tables.